src/components/SheetRenderer.js

```
@@ -0,0 +1,189 @@
+      e.preventDefault();
+    }
+  }
+
+  _onDrop(e) {
+    e.preventDefault();
+    const field = e.dataTransfer.getData('text/plain');
+    if (!field) return;
+
+    const td = e.target.closest('td');
+    if (!td) return;
+    const row = parseInt(td.dataset.r, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)